src/core_plugins/kbn_vislib_vis_types/public/editors/pie.html

```diff
@@ -27,41 +30,53 @@
   <div class="form-group">
     <div class="kuiSideBarSectionTitle">
       <div class="kuiSideBarSectionTitle__text">
-        Labels Settings
+        <span i18n-id="kbnVislibVisTypes.editors.pie.labelsSettings" i18n-default-message="Labels Settings"></span>
```

Comment:
do not create extra `span` tag, use `<div class="kuiSideBarSectionTitle__text">` instead. Also kbnVislibVisTypes.editors.pie.labelsSettings => kbnVislibVisTypes.editors.pie.labelsSettingsTitle